src/styles/blocks/calendar.scss

```
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+  padding: 10px;
+  width: $calendar-width;
+
+  $days:
+    'mon' 0 * $cell-plus-gap,
+    'tue' 1 * $cell-plus-gap,
+    'wed' 2 * $cell-plus-gap,
+    'thu' 3 * $cell-plus-gap,
+    'fri' 4 * $cell-plus-gap,
+    'sat' 5 * $cell-plus-gap,
+    'sun' 6 * $cell-plus-gap;
```

```suggestion
  $days:
    'mon' 0 ,
    'tue' 1,
    'wed' 2,
    'thu' 3,
    'fri' 4,
    'sat' 5,
    'sun' 6;
```
Move it to the variables file and make multiplication inside a loop